### ISSUE TITLE
Not Working Properly On Any Android Device

### ISSUE THREAD

**Issue by pratt1305** [zachsoft/Ionic-Material#63]
I tried on ios device it seems working much perfect but when i m building it in android device it is not rendered properly view are not loaded so is fault for sdk or anything else can't figure out??

**Comment by ukalan** [comment]
even with crosswalk?

**Comment by shaulhadar** [comment]
that's strange, for me bits works great on android....

maybe it would work less good if you have an old phone....

**Comment by maneeshdisodia** [comment]
hey mine device is mi4i android 5. and i tried the demo. its run without any error. but its too slow
please suggest some tweak 

thanks

**Comment by liuwenzhuang** [comment]
Yes, I tried the demo on my android5.1 and it render correct, but bad performace.

**Comment by kimaina** [comment]
have you tried minifying your script? i had the same problem but after using minification it improved performance

**Comment by liuwenzhuang** [comment]
@kimaina How to minify script? 
thanks

**Comment by kimaina** [comment]
@liuwenzhuang 
I used this https://www.npmjs.com/package/ionic-minify
You can also use this http://blog.ionic.io/minifying-your-source-code/ which is much easier

**Comment by liuwenzhuang** [comment]
@kimaina 
Thank you very much. The two help a lot.
